.spelling

```diff
@@ -209,3 +208,136 @@ ArtifactHub
 CryptoKey
 Encrypter
 cmrel
+
+# As per the RFCs, X.509 is the right spelling. x509 and X509 are
```

Comment:
Suggestion: perhaps we could link the relevant RFC here?

Reply:
Good idea; I rephreased and added a link to https://tools.ietf.org/html/rfc5280 👍